.github/workflows/debezium-workflow-pr.yml

```diff
@@ -299,7 +299,7 @@ jobs:
       # Runs each combination concurrently
       matrix:
         profile: [ "mysql-ci", "mysql-ci-gtids", "mysql-ci-percona", "mysql-ci-ssl" ]
-        version-mysql-server: [ "8.0", "8.4" ]
```

Comment:
Would it be sufficient, at least on the upstream CI, to limit this to only 8.0 and 9.0?

Reply:
Well, given the scope of differences I'd prefer to have here both 8.0 and 8.4.